boofuzz/utils/crash_binning.py

```diff
@@ -197,7 +197,7 @@ def export_file(self, file_name):
         self.last_crash = self.pydbg = None
 
         fh = open(file_name, "wb+")
-        fh.write(zlib.compress(pickle.dumps(self, protocol=2)))
+        fh.write(json.dumps(self.bins, default=lambda o: o.__dict__))
```

Comment:
Maybe try another elegant way:

    json.dump(self.bins, open(file_name, "wb+"), default=lambda o: o.__dict__)

Reply:
@cq674350529 That's great. This change is just a merge-in from master, so I'll leave it up to @aedrax if he wants to address it here.

Reply:
I probably won't touch that since it is out of scope of the issue

Reply:
If needed, I can issue another pull request.